multiplatform-markdown-renderer/src/commonMain/kotlin/com/mikepenz/markdown/compose/elements/material/SurfaceWrapper.kt

```diff
@@ -0,0 +1,56 @@
+package com.mikepenz.markdown.compose.elements.material
+
+import androidx.compose.foundation.BorderStroke
+import androidx.compose.foundation.background
+import androidx.compose.foundation.border
+import androidx.compose.foundation.layout.Box
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.draw.shadow
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.RectangleShape
+import androidx.compose.ui.graphics.Shape
+import androidx.compose.ui.input.pointer.pointerInput
+import androidx.compose.ui.semantics.isContainer
+import androidx.compose.ui.semantics.semantics
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.dp
+
+@Composable
+internal fun Surface(
```

Comment:
Is this still needed?

Reply:
To keep it easier to maintain I figured it is best to have the `Surface` logic from Material2 pulled up with some adaptions to not depend on Material themes. 
Making it theoretically simpler to update if Material significantly changes the respective components. 

Would you suggest not having this abstraction?

Reply:
No, although I wonder then if it's not really a material Surface, and rather it's some markdown or mikepenz concept for mobile, that happens to have the same shape as Surface?

For example If I contributed a Wear module, it uses ScalingLazyColumn see https://developer.android.com/training/wearables/compose/lists

So instead of Surface, maybe the Wear the renderer impl splits on paragraphs and renders individually?

Not a blocker.

Reply:
Thank you very much. Appreciate the input!